src/shared/components/person/settings.tsx

```diff
@@ -1175,8 +1175,12 @@ export class Settings extends Component<any, SettingsState> {
       ...i.state.saveUserSettingsForm,
       auth: myAuthRequired(),
     });
+
     if (saveRes.state === "success") {
-      UserService.Instance.login(saveRes.data);
+      UserService.Instance.login({
+        res: saveRes.data,
+        showToast: false,
+      });
```

Comment:
This works for now, but I always found it odd how the flow for changing settings involves making a call to login. Ideally we'd be able to update the settings without doing that.

Reply:
Yeah its def a weird one, should probably be reworked sometime.